sydent/db/threepid_associations.sql

```diff
@@ -19,4 +19,5 @@ CREATE UNIQUE INDEX IF NOT EXISTS medium_address on local_threepid_associations(
 
 CREATE TABLE IF NOT EXISTS global_threepid_associations (id integer primary key, medium varchar(16) not null, address varchar(256) not null, mxid varchar(256) not null, ts integer not null, notBefore bigint not null, notAfter integer not null, originServer varchar(255) not null, originId integer not null, sgAssoc text not null);
 CREATE INDEX IF NOT EXISTS medium_address on global_threepid_associations (medium, address);
+CREATE INDEX IF NOT EXISTS medium_lower_address on global_threepid_associations (medium, lower(address));
```

Comment:
we do need to run this on our deployments: sydent doesn't have a magic schema version-er